docs/en/lab/logger.md

```diff
@@ -1,66 +1,102 @@
-RoadRunner provides the ability to control the log for each plugin individually.
+Logger Plugin is responsible for collecting logs from server plugins and PHP application workers' `STDERR` and
+displaying them in the RoadRunner `STDERR`/`STDOUT`. It comes with a variety of options that allow you to customize the
+way your application logs are collected and displayed.
 
-## Global configuration
+### Configuration
 
-To configure logging globally use `logs` config section:
+### Modes
 
 ```yaml
 logs:
   mode: production
-  output: stderr
 ```
 
-To use develop mode. It enables development mode (which makes DPanicLevel logs panic), uses a console encoder, writes to
-standard error, and disables sampling. Stacktraces are automatically included on logs of WarnLevel and above.
+There are three available modes:
+
+1. `production` - This mode uses logger settings that are optimized for production usage.
+2. `development` - This mode is enabled by default and is designed for use during application development. In
+   development mode, DPanicLevel logs panic, console colors are used, and logs are written to standard error. Sampling
+   is disabled, and stack traces are automatically included on logs of WarnLevel and above.
+3. `raw` - This mode displays messages as raw output without any formatting. This mode is useful in production
+   environments where you need to parse logs programmatically.
+
+> **Note**
+> Use `production` mode in production environments. It is optimized for production usage.
+
+### Encoding
+
+Logger supports two types of encoding, `console` and `json`. By default, `console` encoding is used, which outputs logs
+in a friendly format. JSON encoding, on the other hand, returns messages as a JSON string without formatting. JSON
+encoding is better suited for production usage.
 
 ```yaml
 logs:
-  mode: development
-  output: stderr
+  encoding: console
 ```
 
-To output to [separate file](https://github.com/roadrunner-server/roadrunner/blob/master/.rr.yaml#L115):
+### Level
+
+The level is used to specify the logging level. This means that only log messages with a severity level will be sent to
+this channel. Available levels include `panic`, `error`, `warn`, `info`, and `debug`.
 
 ```yaml
 logs:
-  mode: production
-  output: stderr
-  file_logger_options:
-    log_output: "/tmp/my.log"
-    max_size: 100
-    max_age: 1
-    max_backups : 5
-    compress: false
+  level: info
 ```
 
-To use console friendly output:
+> **Note**
+> The default level is `debug`.
+
+### Output
+
+By default, RoadRunner sends logs to `STDERR`. However, you can configure RoadRunner to send logs to `STDOUT` by using
+the output key.
 
 ```yaml
 logs:
-  encoding: console # default value
+  output: stdout
 ```
 
-To suppress messages under specific log level:
+### Line Endings
+
+It allows configuring custom line endings for the logger. By default, the plugin uses `\n` as the line ending.
```

Comment:
Need to mention, that `\n` is forced default. So, if you specify nothing and expecting to skip `\n`, RR will still use `\n`.